test/d2l-filter-dropdown/d2l-filter-dropdown.js

```diff
@@ -1,387 +1,155 @@
-				afterNextRender(searchButton, function(button) {
-					MockInteractions.tap(button);
-					requestAnimationFrame(function() {
-						var opts = _getOptions(pages[0]);
-						for (var i = 0; i < opts.length; i++) {
-							assert.equal(opts[i].text.indexOf(search) === -1, opts[i].hidden || false);
-						}
-						done();
-					});
-				}, [searchButton]);
-			});
+		test('instantiating the element works', function() {
+			assert.equal('d2l-filter-dropdown-category', categories[0].tagName.toLowerCase());
 		});
-		test('removing filter category updates filters', function() {
-			var expected = _getExpectedAndImport(filter);
-			var removal = categories[0].key;
-			filter.removeFilterCategory(removal);
-			expected = expected.filter(e => e.key !== removal);
-			assert.deepEqual(expected, filter._filters);
+		test('attributes are set correctly (including d2l-tab-panel attributes)', function() {
+			assert.equal(categories[0].key, '1');
+			assert.equal(categories[0].getAttribute('role'), 'tabpanel');
+
+			assert.equal(categories[1].key, '2');
+			assert.equal(categories[1].getAttribute('role'), 'tabpanel');
 		});
-		test('removing filter option updates filters', function() {
-			var expected = _getExpectedAndImport(filter);
-			var removal = [options[1].cat, options[1].key];
-			filter.removeFilterOption(removal[0], removal[1]);
-			var cat = expected.filter(e => e.key === removal[0])[0];
-			cat.options = cat.options.filter(o => o.key !== removal[1]);
-			assert.deepEqual(expected, filter._filters);
+		test('tab text is set properly', function() {
+			assert.equal(categories[0].selectedOptionCount, 3);
+			assert.equal(categories[0].categoryText, 'Category 1');
+			assert.equal(categories[0].text, 'Category 1 (3)');
+
+			assert.equal(categories[1].selectedOptionCount, 0);
+			assert.equal(categories[1].categoryText, 'Category 2');
+			assert.equal(categories[1].text, 'Category 2');
+		});
+		test('search input is hidden if disable-search attribute is present', function() {
+			assert.equal(categories[0].disableSearch, false);
+			var searchInput = categories[0].shadowRoot.querySelector('.d2l-filter-dropdown-page-search');
+			assert.equal(searchInput.hidden, false);
+
+			assert.equal(categories[1].disableSearch, true);
+			searchInput = categories[1].shadowRoot.querySelector('.d2l-filter-dropdown-page-search');
+			assert.equal(searchInput.hidden, true);
 		});
-		test('removing filter category updates display', function(done) {
-			_getExpectedAndImport(filter);
-			var removal = categories[0].key;
-			filter.removeFilterCategory(removal);
-			window.requestAnimationFrame(function() {
-				var headers = _getTabPanels();
-				assert.equal(categories.length - 1, headers.length);
-				assert.deepEqual(categories.filter(c => c.key !== removal).map(c => c.title), [].map.call(headers, h => h.text));
-				done();
-			});
+		test('setting the search-value aattrbiute updates the value in the search input', function() {
```

Comment:
nit: if you do change this file, might as well correct the spelling mistake :P but otherwise don't bother